packages/salesforcedx-vscode-core/package.nls.json

```diff
@@ -4,85 +4,61 @@
+  "force_start_apex_debug_logging": "SFDX: Turn On Apex Debug Log for Replay Debugger",
+  "force_stop_apex_debug_logging": "SFDX: Turn Off Apex Debug Log for Replay Debugger",
 
-  "isv_bootstrap_command_text":
-    "SFDX: Create and Set Up Project for ISV Debugging",
+  "isv_bootstrap_command_text": "SFDX: Create and Set Up Project for ISV Debugging",
   "force_apex_log_get_text": "SFDX: Get Apex Debug Logs...",
 
-  "retrieve_test_code_coverage_text":
-    "Specifies whether code coverage results are calculated and retrieved when you run Apex tests.",
-  "telemetry_enabled_description":
-    "Specifies whether to enable Salesforce telemetry. Even when enabled, still abides by the overall `telemetry.enableTelemetry` vscode setting.",
-  "push_or_deploy_on_save_enabled_description":
-    "Specifies whether or not to automatically run force:source:push (for source-tracked orgs) or force:source:deploy (for non-source-tracked orgs) when a local source file is saved.",
+  "retrieve_test_code_coverage_text": "Specifies whether code coverage results are calculated and retrieved when you run Apex tests.",
+  "telemetry_enabled_description": "Specifies whether to enable Salesforce telemetry. Even when enabled, still abides by the overall `telemetry.enableTelemetry` vscode setting.",
+  "push_or_deploy_on_save_enabled_description": "Specifies whether or not to automatically run force:source:push (for source-tracked orgs) or force:source:deploy (for non-source-tracked orgs) when a local source file is saved.",
   "force_manifest_editor_show_text": "SFDX: Show Package Manifest Editor",
   "force_source_retrieve_text": "SFDX: Retrieve Source from Org",
-  "force_source_retrieve_this_source_text":
-    "SFDX: Retrieve This Source from Org",
-  "force_source_retrieve_in_manifest_text":
-    "SFDX: Retrieve Source in Manifest from Org",
+  "force_source_retrieve_this_source_text": "SFDX: Retrieve This Source from Org",
+  "force_source_retrieve_in_manifest_text": "SFDX: Retrieve Source in Manifest from Org",
   "force_source_deploy_text": "SFDX: Deploy Source to Org",
   "force_source_deploy_this_source_text": "SFDX: Deploy This Source to Org",
-  "force_source_deploy_in_manifest_text":
-    "SFDX: Deploy Source in Manifest to Org",
+  "force_source_deploy_in_manifest_text": "SFDX: Deploy Source in Manifest to Org",
   "force_source_delete_text": "SFDX: Delete from Project and Org",
-  "force_source_delete_this_source_text":
-    "SFDX: Delete This from Project and Org",
-  "force_config_set_org_text": "SFDX: Set a Default Org"
+  "force_source_delete_this_source_text": "SFDX: Delete This from Project and Org",
+  "force_config_set_org_text": "SFDX: Set a Default Org",
+  "initial_sobject_refresh_on_startup_description": "Specifies if SObject definitions should automatically be refreshed on startup if a project has yet to generate them."
```

Comment:
@ruthemmanuelle how is this for the setting description?

Reply:
FYI: Ruth is out today but we can merge the change and edit the label directly on the release branch if needed.